2019-Spring-WashU.md

```diff
@@ -13,7 +13,7 @@ This course is intended to introduce students to the concepts of Open Source sof
 
 Additionally, students should expect to be challenged in areas of personal development related to the business environment such as giving presentations, writing position papers and performing the full lifecycle of software development in a team setting.
 
-### Administrativia
+### Administrativa
```

Comment:
```suggestion
### Administrativia
```
The previous word ([administrativia](https://en.wiktionary.org/wiki/administrativia)) was correct in this context. Please fix this one line so the full PR can be merged.